modules/ROOT/pages/creating-an-odatav4-api-with-apikit.adoc

```diff
@@ -93,22 +93,125 @@ image::odata-query-option-attributes.png["odataRequestAttributes is highlighted
 
 == Use The Transform to SQL Operation
 
-APIkit for OData 4 contains a `Transform to SQL Select` operation to transform the input OData `GET` request to a SQL SELECT query. The operation is useful when combined with the `select` operation of the Database Connector. The output payload is a prepared statement and the attributes are a map of prepared statement parameters.
+APIkit for OData 4 contains a `Transform to SQL Select` operation to transform the input OData `GET` request to a SQL SELECT query. The operation is useful when combined with the `select` operation of the Database Connector. The output payload is a prepared statement and the attributes are a map of prepared statement parameters. It has been designed to work with the Mule DB connector to help the user expose databases with OData 4 easily.
```

Comment:
-"**It** has been designed" > **This operation** has been designed... 
Not completely clear what **it** refers to, please confirm if this suggestion is accurate. 

-to help the user expose databases with OData 4 easily > to simplify the process of exposing databases with OData 4.
This is just a suggestion, consider if you prefer this change or the original.

Reply:
The first comment committed! ;) Thanks!